<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastrousuario</class>
 <widget class="QDialog" name="cadastrousuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>271</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>286</width>
    <height>271</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>258</width>
     <height>248</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineSenha"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Senha</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Usuário</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="botaoSenha">
        <property name="text">
         <string>Alterar Senha</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="botaoNovo">
        <property name="text">
         <string>Criar Usuário</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineUsuario"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableUsuarios"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
